src/sys.zig

```diff
@@ -257,8 +257,8 @@ pub fn getErrno(rc: anytype) bun.C.E {
     const Type = @TypeOf(rc);
 
     return switch (Type) {
-        comptime_int, usize => std.os.linux.getErrno(@as(usize, rc)),
-        i32, c_int, isize => std.os.linux.getErrno(@as(usize, @bitCast(@as(isize, rc)))),
+        comptime_int, usize => return std.os.linux.getErrno(@as(usize, rc)),
```

Comment:
In Zig, the return values of syscalls are all of type `usize`. `libc` use the `c_int`.

https://github.com/ziglang/zig/blob/master/lib/std/os/linux/x86_64.zig
https://github.com/ziglang/zig/blob/master/lib/std/os/linux/arm64.zig